### ISSUE TITLE
There are untagged, merged release PRs outstanding - aborting`

### ISSUE THREAD

**Issue by ddennis** [google-github-actions/release-please-action#656]
### TL;DR

We are using simple subdirectories, but no pull request are made.
We only have a "frontend" and "backend" directories. Not using lerna or any other monorepo tools.
My github action ends with ``There are untagged, merged release PRs outstanding``

My manifest and config file is added under "Additional information"
I am unsure if they are set up correctly.

### Expected behavior

A pull request should be made when new commits are made.
I am using release-please on a couple of other repo's without subdirectories and it is working perfectly.

### Observed behavior

My Github action ends with this:
````
✔ Considering: 458 commits
❯ component:
❯ pull request title pattern: undefined
❯ Found pull request #100: 'chore: release main'
⚠ There are untagged, merged release PRs outstanding - aborting`
````

### Action YAML

```yaml
on:
  push:
    branches:
      - main
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          release-type: node
          package-name: release-please-action
          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":true},{"type":"tests","section":"Tests","hidden":false},{"type":"design","section":"Design/UI","hidden":false},{"type":"text","section":"Texts","hidden":false}]'
```

### Log output

```text
✔ Looking for latest release on branch: main with prefix: release-please-action
❯ Fetching merge commits on branch main with cursor: undefined
❯ Fetching merge commits on branch main with cursor: 2bbbadcbe906a674d66a7a268b9e22fc686139cf 24

...... additionaly "Fetching merge commits on" removed for readability 
        
❯ Fetching merge commits on branch main with cursor: 2bbbadcbe906a674d66a7a268b9e22fc686139cf 224
❯ Fetching releases with cursor undefined
❯ found 0 possible releases. []
❯ found 0 possible tags. []
✔ Building releases
✔ Building strategies by path
❯ .: node
❯ Found pull request #100: 'chore: release main'
✔ Building release for path: .
❯ type: node
❯ targetBranch: main
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
✔ Pull request contains releases, but not for component: release-please-action
✔ No release necessary for path: .
✔ Looking for latest release on branch: main with prefix: release-please-action
❯ Fetching merge commits on branch main with cursor: undefined
❯ Fetching merge commits on branch main with cursor: 2bbbadcbe906a674d66a7a268b9e22fc686139cf 24
...... additionaly "Fetching merge commits on" removed for readability
❯ Fetching merge commits on branch main with cursor: 2bbbadcbe906a674d66a7a268b9e22fc686139cf 224
❯ Fetching releases with cursor undefined
❯ found 0 possible releases. []
❯ found 0 possible tags. []
✔ Building pull requests
✔ Building strategies by path
❯ .: node
✔ Collecting release commit SHAs
❯ release search depth: 400
❯ Fetching releases with cursor undefined
⚠ Found release tag with component 'recruitment-admin-frontend', but not configured in manifest
⚠ Found release tag with component 'recruitment-backend', but not configured in manifest
⚠ Expected 1 releases, only found 0
⚠ Missing 1 paths: .
⚠ No version for path .
⚠ Expected 1 releases, only found 0
✔ Collecting commits since all latest releases
❯ commit search depth: 500
❯ Set(0) {}
❯ Fetching merge commits on branch main with cursor: undefined
❯ Backfilling file list for commit: dcbe906a674d66a7a268b9e22fc686139cf

...... additionaly "Backfilling file list for commit " removed for readability

❯ Found 1 files
❯ Backfilling file list for commit: e0eb083c6023ffa745bf089c945daff682
❯ Found 1 files
✔ Splitting 500 commits by path
✔ Building candidate release pull request for path: .
⚠ No latest release pull request found.
❯ type: node
❯ targetBranch: main
❯ commits: 500
❯ commit could not be parsed: af310627a70bd1fcebcb582d67d0055e270e1952 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 2b0f15863e506dda18f8d81621ecdbd84f022b83 Merge pull request #100 from StudiesAndMe/release-please--branches--main
❯ commit could not be parsed: f5b6beaa13b7b0dcdb936c8a3dd59e4a97fd5659 chore. testing
❯ commit could not be parsed: c7233ae92ec1063c109735159dcf0fba54ee22a0 Merge pull request #99 from StudiesAndMe/release-please/bootstrap/default
❯ commit could not be parsed: 268ebbe45480c873c27c1b3e3f4aefdc384bc829 Merge pull request #97 from StudiesAndMe/release-please--branches--main
❯ commit could not be parsed: de8c202e9f2c44f697b40bed2d4031632361ab43 Merge pull request #96 from StudiesAndMe/release-please/bootstrap/default
❯ commit could not be parsed: 1f8e57d855dc0da8e3172855b0bea8e27e9a9f2b Merge pull request #95 from StudiesAndMe/staging
❯ commit could not be parsed: a093633e40e51f2192d8559e863bcbdf297cc256 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 1c7ee598cc3449750b83ea04242495da712395f5 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: b1b7f337400e2079ea59e839afe9cbe9d69b6b01 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 1bae54a554557e03b22816ab96683a2e0f659f6c Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin
❯ commit could not be parsed: bd4c24e876dc78f5fd0ccfd9ebb176b71020af22 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 30d69de650d7f9e5de0dda1e82a2c367140064d8 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: f02a4552b07eafe005e88815142f2f6393e43545 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin
❯ commit could not be parsed: 8f131c0cee915f20b0cf575e9c012d9c764fe16f Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 8c587aabf97e3b0e2a42dc056f1db6d0b210885f Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin
❯ commit could not be parsed: 4d1eea2f0a0eeb543920c62b26ef4025ad23e7a5 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: da69a3a21441efccb2ab2b755c50fd5e7ac54f8d Merge pull request #87 from StudiesAndMe/unsubscribe-logic
❯ commit could not be parsed: fc38ac6f1348151cbed8b9c1fc2cebe4c0f3ce61 Merge branch 'main' into unsubscribe-logic
❯ commit could not be parsed: 5831ca3ffd187c9e31f283899c1a1f711ac16b59 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: f8ced21195008f890c8dd0ff325113d313686cae Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin
❯ commit could not be parsed: cab497a071189c067c6c1384903055d54a6c9c8e Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 104e0b9d03e219c36d38128a84cc6fe5a497e03a Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 29fc62068ae4adeca94398fd3a0cf47737f12736 fix (getNextQuestion):  the code assumed that if i had a questionnaire i had also answered a question.
❯ commit could not be parsed: 922df2b39e630a7340f31474cd481052a496c1ad Merge pull request #81 from StudiesAndMe/cleanup
❯ commit could not be parsed: b17c41c74b46b3a6896b44bee5c1c467607ab50e Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: b32a84854eff6fa39e5feac22d50f81706123c6b Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: a7b526e75d742284eb5df821fdbae28e3c681547 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 2c27d42d79434ca2161d145beea7cfd5715c4ef8 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 12a2b8e23e0ce48bd3e3c4f72e37f287fca5d5e2 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 2bc076839a30558535f9e9e26189c8c653040b72 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: e8ab691e8d6bb18552a4dcce52e4bd97c0586629 Merge branch 'retention-branch' of https://github.com/StudiesAndMe/recruitment-admin into retention-branch
❯ commit could not be parsed: 07a25dd5723adbc0ee8e9b0d8f2fc19ac22b12c7 Merge pull request #67 from StudiesAndMe/retention-branch
❯ commit could not be parsed: 982d89511c2fd004ab9d38a2181a70d4977d4095 Merge branch 'main' into retention-branch
❯ commit could not be parsed: 76712d53f459b1c3d204c4ca7863e3f05ad6fc17 Merge branch 'main' into retention-branch
❯ commit could not be parsed: 90b2e8646f59901f76dc9997beb0c16d080bf131 refactor sitetool WIP
❯ commit could not be parsed: 0b7344dc5feedee3659c41fe6af2acc9518ac1f7 refactor. allow custom error messges to be defined and shown with formhook
❯ commit could not be parsed: 04e19c40d612070ed5d6d2d9729ab83b3c05a420 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: 673b686fa80984ed29c732391f453b918c93b642 Merge branch 'main' of https://github.com/StudiesAndMe/recruitment-admin into main
❯ commit could not be parsed: a8bf3c279e445539713f79ff41dd68fce7433550 Merge pull request #40 from StudiesAndMe/mail-temp
✔ Considering: 460 commits
❯ component: release-please-action
❯ pull request title pattern: undefined
❯ Found pull request #100: 'chore: release main'
⚠ There are untagged, merged release PRs outstanding - aborting
```

### Additional information

My manifest and config files looks like this:
````
.release-please-manifest:
{
  "backend": "1.1.0",
  "frontend": "1.1.0"
}

release-please-config:

{
  "packages": {
    "bootstrap-sha": "8ec612df76d56c6515a49b69477c41ea66604770",
    "release-type": "node",
    "backend": {
      "changelog-path": "CHANGELOG.md",
      "bump-minor-pre-major": false,
      "bump-patch-for-minor-pre-major": false,
      "draft": false,
      "release-type": "node",
      "prerelease": false,
      "package-name": "package.json"
    },
    "frontend": {
      "changelog-path": "CHANGELOG.md",
      "bump-minor-pre-major": false,
      "bump-patch-for-minor-pre-major": false,
      "draft": false,
      "release-type": "node",
      "prerelease": false,
      "package-name": "package.json"
    }
  }
}
````

I found this closed issue: https://github.com/google-github-actions/release-please-action/issues/443

**Comment by ddennis** [comment]
After updating my action .yml to use ``command: manifest``

https://github.com/marketplace/actions/release-please-action
````
on:
  push:
    branches:
      - main
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          command: manifest
          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":true},{"type":"tests","section":"Tests","hidden":false},{"type":"design","section":"Design/UI","hidden":false},{"type":"text","section":"Texts","hidden":false}]'

````

The github action is now ending  with:
````
release-please failed: node (StudiesAndMe/recruitment-admin): Missing required file: bootstrap-sha/package.json
````

**Comment by ddennis** [comment]
I got it to work with following **release-please-config**.
But when removing the top level **bootstrap-sha** i started getting the API rate limit issue again. 

````
release-please failed: API rate limit exceeded for installation ID 11998081.
````

````
{
  "bootstrap-sha": "******2df76d56c6515a49b69477c41ea666****",
  "draft": true,
  "release-type": "node",
  "packages": {

    "backend": {
      "changelog-path": "CHANGELOG.md",
      "bump-minor-pre-major": true,
      "bump-patch-for-minor-pre-major": true,
      "draft": false,
      "prerelease": false,
      "package-name": "recruitment-backend"
    },
    "frontend": {
      "changelog-path": "CHANGELOG.md",
      "bump-minor-pre-major": true,
      "bump-patch-for-minor-pre-major": true,
      "draft": false,
      "prerelease": false,
      "package-name": "recruitment-admin-frontend"
    }
  }
}
````

I am clearly just learning this, but the format or the error messages was not clear to me.
````
 Missing required file: draft-pull-request/package.json
 Missing required file: bootstrap-sha/package.json
````

Fixing the this, means you need to add the that key into the **release-please-config**. But when reading the docs i did not understand that the values was required.

**Comment by chingor13** [comment]
Sorry for the delay in getting to this.

The missing required file error was because the original configuration had `bootstrap-sha` and `release-type` under `packages` which is not where those configs go. The keys under `packages` are paths to releaseable components -- release-please was looking for a path `bootstrap-sha/package.json` because it assumed `bootstrap-sha` was a node package to release (and `package.json` is a required file for a node package).

Is the original issue still happening?
